src/cascadia/TerminalSettingsEditor/MainPage.h

```diff
@@ -43,8 +43,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         void _Navigate(hstring clickedItemTag);
         void _Navigate(const Editor::ProfileViewModel& profile);
 
-        ColorSchemesPageNavigationState _colorSchemesNavState{ nullptr };
-        ProfilePageNavigationState _lastProfilesNavState{ nullptr };
+        winrt::Microsoft::Terminal::Settings::Editor::ColorSchemesPageNavigationState _colorSchemesNavState{ nullptr };
+        winrt::Microsoft::Terminal::Settings::Editor::ProfilePageNavigationState _lastProfilesNavState{ nullptr };
```

Comment:
We could also just write `Editor::...`.
Since C++ looks up typenames recursively in the current namespace, `Editor::ColorSchemesPageNavigationState` should correctly resolve to the proper type name.

Reply:
Looking at other files, just using `Editor::` seems like an outlier to me here so using it here doesn't seem right for me. If using the proposed access is actually the usual way of doing this, let me know.

Reply:
I'm fine with it either way. :)